### PR TITLE
Schedule for event added and multiple selection in navigation bar resolved. Issue #180 & Issue #207.

### PR DESCRIPTION
**Pull request by Mudit18**
Now only one option changes to orange in one selection.

### CODE REVIEW

**Comment by mananwason** [comment]
What issues are you solving here @Mudit18? Not clear.

File: app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleListAdapter extends BaseRVAdapter<Event,ViewHolder.Viewholder> {
+
+    public ScheduleListAdapter(List<Event> eventsList) {
+        super(eventsList);
+    }
+
+    private ViewHolder.SetOnClickListener listener;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java]: Codacy Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://www.codacy.com/app/triplez/open-event-android/file/2406791376/issues/source?bid=3233850&fileBranchId=3233907#l27)

File: app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
+package org.fossasia.openevent.adapters;
+
+import android.graphics.Color;
+import android.util.Log;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java]: Codacy Issue found: [Avoid unused imports such as 'android.util.Log'](https://www.codacy.com/app/triplez/open-event-android/file/2406791376/issues/source?bid=3233850&fileBranchId=3233907#l4)

File: app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import org.fossasia.openevent.R;
+import org.fossasia.openevent.data.Event;
+import org.fossasia.openevent.data.Session;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/adapters/ScheduleListAdapter.java]: Codacy Issue found: [Avoid unused imports such as 'org.fossasia.openevent.data.Session'](https://www.codacy.com/app/triplez/open-event-android/file/2406791376/issues/source?bid=3233850&fileBranchId=3233907#l13)

File: app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import org.fossasia.openevent.data.Event;
+
+import java.util.ArrayList;
+
+/**
+ * Created by md91829 on 3/3/16.
+ */
+public class ScheduleFragment extends Fragment {
+
+    private RecyclerView recyclerView;
```
Comment (juslee) [inline app/src/main/java/org/fossasia/openevent/fragments/ScheduleFragment.java]: Codacy Issue found: [Perhaps 'recyclerView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406791428/issues/source?bid=3233850&fileBranchId=3233907#l23)

**Pull request closed without merging**